### PR TITLE
Add details place fetch

### PR DESCRIPTION
**Pull request by ebidel**
R: @brendankenny

### CODE REVIEW

File: google-map-search.html [hunk 1]
```diff
@@ -166,6 +173,21 @@
       }
     },
 
+    /**
+     * Fetches details for a given place.
+     * @param {String} placeId The place id.
```
Comment (brendankenny) [inline google-map-search.html]: lowercase `string`?

File: google-map-search.html [hunk 2]
```diff
@@ -166,6 +173,21 @@
...
+     * @return {Promise} place The place information.
+     */
+    getDetails: function(placeId) {
+      var places = new google.maps.places.PlacesService(this.map);
```
Comment (brendankenny) [inline google-map-search.html]: do you want to cache the `PlacesService` after creation? Not sure of the overhead for them.

File: google-map-search.html [hunk 3]
```diff
@@ -166,6 +173,21 @@
...
+      return new Promise(function(resolve, reject) {
+        places.getDetails({placeId: placeId}, function(place, status) {
+          resolve(place);
```
Comment (brendankenny) [inline google-map-search.html]: if you check `status` you can also reject here a la promisifying XHRs

**Comment by brendankenny** [comment]
reviewed

File: google-map-search.html (hunk 1 above)
Comment (ebidel) [inline google-map-search.html]: No. Not for the docs

Comment (ebidel) [inline google-map-search.html]: https://github.com/Polymer/polymer/blob/8134fbf640c085cb9de8e53ae3353bb3c2132652/src/micro/debouncer.html#L32

File: google-map-search.html (hunk 2 above)
Comment (ebidel) [inline google-map-search.html]: Seems overkill

File: google-map-search.html (hunk 3 above)
Comment (ebidel) [inline google-map-search.html]: Done

**Pull request merged**